_includes/footer.html

```diff
@@ -1,10 +1,10 @@
 <footer class="usa-footer site-footer" role="contentinfo">
-  <div class="footer-section-bottom bg-base-lightest">
+  <div class="footer-section-bottom bg-base-darker">
```

Comment:
Should we make this a site-wide color change? Maybe consult a designer?

Reply:
This will be a site-wide change :)